js/src/components/query/QueryDiffResultView.tsx

```diff
@@ -70,10 +76,15 @@ export const QueryDiffResultView = ({
       }
     };
 
+    const handleInvalidPrimaryKey = (base: string[], current: string[]) => {
```

Comment:
Please use the return value rather than a callback. 

Reason: the invalid primary key is resulted from the input `options.primaryKeys`, but not user's action.